docker-compose.yml

```diff
@@ -25,6 +27,29 @@ services:
     command: uvicorn main:app --host 0.0.0.0 --reload
     networks:
       - microservices
+  permissions-database:
+    container_name: permissions-database
+    networks:
+      - microservices
+    image: postgres
+    expose:
+      - ${DATABASE_PORT}
+    ports:
+      - ${DATABASE_PORT}:${DATABASE_PORT}
+    command: -p ${DATABASE_PORT}
+    volumes:
+      - permissions-database:/var/lib/postgresql/data
+      - ./init.sql:/init.sql
+    hostname: ${DATABASE_HOST}
+    environment:
+      - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
```

Comment:
Call this variables the same name and just use .env_file

Reply:
.env file needs to be in the same folder that is docker-compose. https://docs.docker.com/compose/env-file/